plugins/carbon_black_defense/komand_carbon_black_defense/util/util.py

```diff
@@ -0,0 +1,28 @@
+from functools import wraps
+
+from _datetime import datetime
+
+
+class Util:
+    def retry(tries: int, timeout: int, exceptions):
```

Comment:
It may be a good idea to use an exponential backoff here to prevent possible rate limiting from the API. If you get into a rate limiting loop, it will never succeed. You also have to keep in mind that users may be using this plugin in multiple workflows, which means there could be two workflows hitting the API simultaneously (increasing the risk of rate limiting).

Reply:
Have updated now with an exponential backoff.  @mrinehart-r7 Do we have a standardised method or algorithm for this?